README.md

```diff
@@ -38,6 +38,23 @@ It can then be invoked as:
 BEAKER_setfile=centos7-64 bundle exec rake beaker
 ```
 
+known working setfiles that we currently (2022-01-06) use:
+
+* ubuntu1604-64
+* ubuntu1804-64
+* ubuntu2004-64
+* ubuntu2204-64
+* debian9-64
+* debian10-64
+* debian11-64
+* centos7-64
+* centos8-64
+* centos9-64
```

Comment:
Should we mention that this is really CentOS Stream?

Reply:
good idea. updated it